docs/architecture/009-review-and-form-model.md

```diff
@@ -0,0 +1,93 @@
+# 009: Review and form model
+
+Authors: @macroz
+
+## Background
+
+In the existing handling process, the reviewer has been thought of as having the same viewing rights as the handler. It has been assumed that all the people that need to take part in the handling can and should see the whole application. However, now we have a new need to limit the visibility of fields to reviewers.
+
+First step we took was to include per field privacy state: `public` or `private`. Public fields are shown to all the people who take part in handling, including reviewers. Private fields are not sent to reviewers and are meant for the handler and decider only.
+
+While this solves the immediate problem of not letting everyone see everything, ideally some fields could be public for some reviewer and private for another. Now the information from a private field must be copy-pasted to the review request comment or shared otherwise.
+
+## Options
+
+There exist at least six possibilities to solve this.
+
+1. Require the handler to copy-paste the relevant data from private fields to each reviewer (status quo).
+   Application contains form fields of one form.
+2. Model each reviewer organization as a catalogue item with extra fields.
+   Application contains form fields of one form and all extra fields from all catalogue items.
+   Review request is sent with certain catalogue items selected and it controls which fields are shown to the reviewer.
+3. Model each reviewer organization as a catalogue item with a separate form.
+   Application contains form fields from all forms of all catalogue items.
+   Review request is sent with certain catalogue items selected and it controls which fields are shown to the reviewer.
+4. Model each reviewer organization as a catalogue item with same form.
+   Model private fields as repeating fields that repeat per catalogue item.
+   Application contains form fields of one form.
+   Review request is sent with certain catalogue items selected and it controls which fields are shown to the reviewer.
+5. Add a field type to the model where you can add multiple answers.
+   Model private fields as repeating fields that repeat per each answer to the multiple answer field.
```

Comment:
I think I understand option 5 but I think the description could be clearer

Reply:
Same as above.